### ISSUE TITLE
Windows 10 tox error: SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 2-3: truncated \UXXXXXXXX escape

### ISSUE THREAD

**Issue by thernstig** [PyCQA/pylint#1930]
### Steps to reproduce
1. Use Windows 10 (64 bit)
2. Remove/comment `pyenchant` from `tox.ini` (since pyenchant does not work on Windows 10, see #1900
3. Run tox

### Current behavior
```python
============================ 248 tests deselected =============================
========== 857 passed, 107 skipped, 248 deselected in 78.78 seconds ===========
Coverage.py warning: No data was collected. (no-data-collected)
py36 runtests: commands[1] | python -c import os;cov_strip_abspath = open(os.environ['COVERAGE_FILE'], 'r').read().replace('.tox' + os.sep + os.path.relpath('C:\Users\Moo\Desktop\projects\pylint\.tox\py36\Lib\site-packages', 'C:\Users\Moo\Desktop\projects\pylint\.tox') + os.sep, '');open(os.environ['COVERAGE_FILE'], 'w').write(cov_strip_abspath)
  File "<string>", line 1
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 2-3: truncated \UXXXXXXXX escape
ERROR: InvocationError: "C:\\Users\\Moo\\Desktop\\projects\\pylint\\.tox\\py36\\Scripts\\python.EXE -c import os;cov_strip_abspath = open(os.environ['COVERAGE_FILE'], 'r').read().replace('.tox' + os.sep + os.path.relpath('C:\\Users\\Moo\\Desktop\\projects\\pylint\\.tox\\py36\\Lib\\site-packages', 'C:\\Users\\Moo\\Desktop\\projects\\pylint\\.tox') + os.sep, '');open(os.environ['COVERAGE_FILE'], 'w').write(cov_strip_abspath)"
___________________________________ summary ___________________________________
ERROR:   py36: commands failed
~\Desktop\projects\pylint [bug/epylint_windows_rcfile]> git st
```

### Expected behavior
That tox should run properly

### pylint --version output
pylint 1.7.1
python version 3.6.1

**Comment by brycepg** [comment]
What version is tox (`tox --version`)?

**Comment by thernstig** [comment]
@brycepg I edited the original post with the version.

**Comment by brycepg** [comment]
I'm unable to reproduce this using

```
pylint 2.0.0
astroid 1.6.1
Python 3.6.5rc1 (v3.6.5rc1:f03c5148cf, Mar 14 2018, 03:12:11) [MSC v.1913 64 bit (AMD64)]
tox 2.9.1
```
On Windows 10 64 bit

**Comment by thernstig** [comment]
To make sure, I tried again and could reproduce it on the versions  Could it possibly be a problem that a raw string is not used? See https://stackoverflow.com/questions/18084554/why-do-i-get-a-syntaxerror-for-a-unicode-escape-in-my-file-path
